staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/cel_validation.go

```diff
@@ -70,37 +72,33 @@ type CELSchemaContext struct {
 	TotalCost *TotalCost
 }
 
-// HasValidationRules returns true if the current schema node contains x-kubernetes-validations rules.
-func (c *CELSchemaContext) HasValidationRules() bool {
-	return len(c.jsonSchema.XValidations) > 0
-}
-
 // TypeInfo returns the CELTypeInfo for this CELSchemaContext node.  Returns nil if current level or above does not have
```

Comment:
nit: suggest "... Returns nil, nil if this CELSchemaContext is nil, or if the current level or above..."